jenkins-integration/jenkins-jobs/scenarios/oss-puppetserver-latest-jruby9k-json-vs-pson/Jenkinsfile

```diff
@@ -0,0 +1,57 @@
+node {
+    checkout scm
+    pipeline = load 'jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy'
+}
+
+pipeline.multipass_pipeline([
+        [
+                job_name: 'oss-latest-jruby-9k-json',
+                gatling_simulation_config: '../simulation-runner/config/scenarios/foss5x-medium-1250-2-hours.json',
+                server_version: [
+                        type: "oss",
+                        version: "latest"
+                ],
+                agent_version: [
+                        version: "latest"
+                ],
+                code_deploy: [
+                        type: "r10k",
+                        control_repo: "git@github.com:puppetlabs/puppetlabs-puppetserver_perf_control.git",
+                        basedir: "/etc/puppetlabs/code/environments",
+                        environments: ["production"],
+                        hiera_config_source_file: "/etc/puppetlabs/code/environments/production/root_files/hiera.yaml"
+                ],
+                background_scripts: [
+                        "./jenkins-jobs/common/scripts/background/curl-server-metrics-loop.sh"
+                ],
+                archive_sut_files: [
+                        "/var/log/puppetlabs/puppetserver/metrics.json"
+                ],
+                jruby_jar: "/opt/puppetlabs/server/apps/puppetserver/jruby-9k.jar",
+        ],
+        [
+                job_name: 'oss-latest-jruby9k-pson',
+                gatling_simulation_config: '../simulation-runner/config/scenarios/foss25x-medium-1250-2-hours.json',
```

Comment:
Belated answer on this one...  The main thing we're looking for in this comparison is how the use of PSON serialization compares to the use of JSON serialization in agent->master communication.  By default, a Puppet 5 agent will try use to use JSON.  We could do a re-recording of a Puppet 5 agent where the preferred serialization format is set to PSON, but effectively the existing FOSS 25x already gets us PSON by default.  FOSS25x was captured from a Puppet 4 agent run.  Since Puppet 5 is backward compatible with Puppet 4 agents on the wire, I think just reusing the existing FOSS25x recording is reasonable here.